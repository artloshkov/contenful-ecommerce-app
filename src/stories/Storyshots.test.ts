import initStoryshots from "@storybook/addon-storyshots";
import { imageSnapshot } from "@storybook/addon-storyshots-puppeteer";
import "jest-styled-components";

initStoryshots({ suite: "Puppeteer storyshots", test: imageSnapshot({ storybookUrl: "http://localhost:6006/" }) });
