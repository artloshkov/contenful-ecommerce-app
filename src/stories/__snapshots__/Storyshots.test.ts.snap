// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Example/CartIcon Cart Icon Template 1`] = `
.c0 {
  padding: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  box-sizing: border-box;
  font-family: -apple-system,BlinkMacSystemFont,"Segoe UI","Roboto","Oxygen","Ubuntu","Cantarell","Fira Sans","Droid Sans","Helvetica Neue",sans-serif;
}

.c0 span {
  font-weight: 600;
  margin-right: 0.5rem;
  background: #2b2b2b;
  color: #fff;
  font-size: 1rem;
  min-width: 32px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  text-align: center;
  height: 32px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 svg {
  font-size: 1.5rem;
  color: #2b2b2b;
}

<a
  className="c0"
  href="/cart"
  onClick={[Function]}
>
  <span>
    5
  </span>
  <svg
    aria-hidden="true"
    className="svg-inline--fa fa-shopping-cart fa-w-18 "
    data-icon="shopping-cart"
    data-prefix="fas"
    focusable="false"
    role="img"
    style={Object {}}
    viewBox="0 0 576 512"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M528.12 301.319l47.273-208C578.806 78.301 567.391 64 551.99 64H159.208l-9.166-44.81C147.758 8.021 137.93 0 126.529 0H24C10.745 0 0 10.745 0 24v16c0 13.255 10.745 24 24 24h69.883l70.248 343.435C147.325 417.1 136 435.222 136 456c0 30.928 25.072 56 56 56s56-25.072 56-56c0-15.674-6.447-29.835-16.824-40h209.647C430.447 426.165 424 440.326 424 456c0 30.928 25.072 56 56 56s56-25.072 56-56c0-22.172-12.888-41.332-31.579-50.405l5.517-24.276c3.413-15.018-8.002-29.319-23.403-29.319H218.117l-6.545-32h293.145c11.206 0 20.92-7.754 23.403-18.681z"
      fill="currentColor"
      style={Object {}}
    />
  </svg>
</a>
`;

exports[`Storyshots Example/ModalCloseButton Modal Close Button Template 1`] = `
.c0 {
  line-height: 1;
  box-sizing: border-box;
}

.c0 span {
  font-size: 2rem;
}

<button
  className="close c0 btn"
  onClick={[Function]}
  type="button"
>
  <span
    aria-hidden="true"
  >
    Ã—
  </span>
  <span
    className="sr-only"
  >
    Close
  </span>
</button>
`;

exports[`Storyshots Example/PageHeading Page Heading Template 1`] = `
.c0 {
  font-family: -apple-system,BlinkMacSystemFont,"Segoe UI","Roboto","Oxygen","Ubuntu","Cantarell","Fira Sans","Droid Sans","Helvetica Neue",sans-serif;
  text-align: center;
  padding: 1rem 0;
  line-height: 1.25;
  background-color: #eee;
  margin: 1rem 0;
  font-size: calc(1.375rem + 1.5vw);
  font-weight: 500;
}

<h1
  className="c0"
>
  Test Heading
</h1>
`;
